go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869
 	github.com/fsnotify/fsnotify v1.4.9
 	github.com/mreiferson/go-options v1.0.0
+	github.com/openshift/build-machinery-go v0.0.0-20200819073603-48aa266c95f7
```

Comment:
nit: this technically goes to bump commit

Reply:
I'd keep it here as "adding a new dependency due to changes in the repo", I'm keeping the vendoring in a separate commit though

Reply:
ok, we don't enforce it yet, but be prepared for "every change to go.{mod,sum} or vendor/" must start with "bump:" message